en/gps_compass/README.md

```diff
@@ -50,6 +50,8 @@ Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [GPS Yaw Output](#co
 [Holybro H-RTK M8P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-m8p.md) | M8P | IST8310 | &check; | | |
 [Hobbyking u-blox Neo-M8N GPS with Compass](https://hobbyking.com/en_us/ublox-neo-m8n-gps-with-compass.html?gclid=Cj0KCQjwqM3VBRCwARIsAKcekb3ojv1ZhLz1-GuvCsUuGT8ZZuw8meMIV_I6pgUCj6DJRzHBY9OApekaAgI5EALw_wcB&gclsrc=aw.ds&___store=en_us) | M8N | &check; | | | |
 [LOCOSYS Hawk A1 GNSS receiver](../gps_compass/gps_locosys_hawk_a1.md) | MC-1612-V2b | optional | &cross; | | | 
+[LOCOSYS Hawk R1](../gps_compass/gps_locosys_hawk_r1.md) | MC-1612-V2b |  | &cross; | | | 
+[LOCOSYS Hawk R2](../gps_compass/gps_locosys_hawk_r2.md) | MC-1612-V2b | ? | &cross; | | | 
```

Comment:
What is the Compass part on the Hawk R2? Note, I removed "optional" from H1.

Reply:
They were planning to release only R1 at that time and integrate compass if needed. Now they have changed their minds and started manufacturing R2 with an integrated compass already.

Reply:
Yes, but what is the "part" number. Almost all compasses are standard chips like "IST8310".

I'll review again when you've answered the comments

Reply:
yes, was IST8310. Corrected.